src/__test__/index.test.ts

```diff
@@ -1,5 +1,6 @@
 import { params, fragment, query, mutation, types, optional, alias, on, rawString } from '../index'
```

Comment:
Could you import `onUnion` from here, instead of `app/types`?

Reply:
So that we can do `import { onUnion } from 'typed-graphqlify'`, currently `src/index.ts` exports them

Reply:
yep, sorry about that one. I must have used the vscode auto-import and not double checked it